pages/interview-process.md

```diff
@@ -6,7 +6,7 @@ title: Interview process
 
 ### Phone screen
 
-Apply to 18F by emailing your resume to [join18f@gsa.gov](mailto:join18f@gsa.gov). Within three weeks our team will review your resume, and if you seem like a good fit, we'll reach out to schedule a 30-minute phone call. Plan to talk about your skills and experience, what you're passionate about, and the work we do here. 
+[Apply to 18F](/how-to-apply). Within three weeks our team will review your resume, and if you seem like a good fit, we'll reach out to schedule a 30-minute phone call. Plan to talk about your skills and experience, what you're passionate about, and the work we do here. 
```

Comment:
According to [this][1], the link needs to be prefixed with `{{ site.baseurl }}`.

[1]: https://pages.18f.gov/guides-template/add-a-new-page/